packages/email-plugin/src/plugin.ts

```diff
@@ -91,6 +116,14 @@ import {
  * `node_modules/\@vendure/email-plugin/templates` to a location of your choice, and then point the `templatePath` config
  * property at that directory.
  *
+ * * ### Dynamic Email Templates
+ * Instead of passing a static value to `templatePath`, use `templateLoader` to define a template path.
+ * ```ts
+ *   EmailPlugin.init({
+ *    ...,
+ *    templateLoader: new FileBasedTemplateLoader(my/order-confirmation/template)
```

Comment:
The path should be a string, and also `my/order-confirmation/template` is a bit confusing because as far as I understand, this path should be the directory that houses _all_ the templates, rather than just 1 type as suggested by "order-confirmation". Am I understanding this correctly?